api/app/controllers/user.controller.ts

```diff
@@ -42,6 +53,53 @@ export async function create(req: Request, res: Response) {
   }
 }
 
+// Sign in a user
+export async function signIn(req: Request, res: Response) {
+  try {
+    // Validate request
+    if (!req.body.oauthToken) {
+      return res.status(400).send({
+        message: "OAuth token can not be empty!",
+      });
+    }
+
+    const { oauthToken } = req.body;
```

Comment:
In signup it is request.body.githubAuthToken, here it is oauthToken.
For standardization, let's use the same name. Is there a specific reson for the segregation?

Reply:
Sure can change that. @Extheoisah any opinion

Reply:
I agree.